unlock-app/src/__tests__/components/helpers/Balance.test.js

```diff
@@ -25,10 +35,11 @@ describe('Balance Component', () => {
     const wrapper = render(<Balance
       amount={amount}
       unit={unit}
+      conversion={conversion}
     />)
 
     it('shows the default minimum value of 三 < 0.0001', () => {
-      expect(wrapper.text()).toEqual('三 < 0.0001')
+      expect(wrapper.text()).toEqual('< 0.00010.014')
```

Comment:
Sam here as above. Maybe this time instead of checking for the text value we check for the actual markup? (and I start to see where your comments about Enzyme are coming from!)

Reply:
fixed